ios/Podfile

```diff
@@ -24,7 +24,7 @@ target 'Backpack Native' do
   pod 'Folly', podspec: '../node_modules/react-native/third-party-podspecs/Folly.podspec'
 
   # RN Bridging Pods
-  pod 'Backpack', '~> 6.3.0'
+  pod 'Backpack', '~> 7.0.0'
```

Comment:
May as well bump this to 8.0.0 now that's out.

Reply:
Yeah shounds good